src/main/java/net/minecraftforge/coremod/api/ASMAPI.java

```diff
@@ -24,15 +24,31 @@
 import java.util.Objects;
 import java.util.Optional;
 
+/**
+ * Helper methods for working with ASM.
+ *
+ * @author cpw
+ */
 public class ASMAPI {
+    // Why do we have this? At this point, you should just import MethodNode yourself or use a MethodVisitor.
     public static MethodNode getMethodNode() {
```

Comment:
Its because ASMAPI is intended to be consumed by js code, and imports in js are annoying.

Reply:
ahh ok. then honestly in #49 I might add a few methods like this where I otherwise would've needed imports. Thanks.